packages/apify/src/configuration.ts

```diff
@@ -185,6 +189,7 @@ export class Configuration extends CoreConfiguration {
         standbyPort: +LOCAL_ACTOR_ENV_VARS[ACTOR_ENV_VARS.STANDBY_PORT],
         metamorphAfterSleepMillis: 300e3,
         persistStateIntervalMillis: 60e3, // This value is mentioned in jsdoc in `events.js`, if you update it here, update it there too.
+        testPayPerEvent: false,
```

Comment:
do we have this documented somewhere? how does the testing work?

Reply:
Same place as the rest of PPE support in the SDK - in the future.

You're supposed to set the `ACTOR_TEST_PAY_PER_EVENT` environment variable when developing locally to put the ChargingManager into PPE pricing mode. If you don't do that, it will only log a warning when you call `Actor.charge` or something.